### PR TITLE
Display resubmission author and runtime environment on ReferenceSolut…

### PR DESCRIPTION
**Pull request by Neloop**
…ionDetail page

### CODE REVIEW

File: src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js [hunk 1]
```diff
@@ -59,18 +63,46 @@ const ReferenceSolutionStatus = ({ description, solution: { userId, createdAt },
...
+            <th className="text-nowrap">
+              <FormattedMessage id="app.referenceSolution.reevaluatedBy" defaultMessage="Re-evaluated by" />:
+            </th>
+            <td>
+              <UsersNameContainer userId={submittedBy} showEmail="icon" />
+            </td>
+          </tr>
+        )}
+
+        {Boolean(environment) && Boolean(environment.name) && (
```
Comment (SemaiCZE) [inline src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js]: Why checking `environment.name` here?

File: src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js [hunk 2]
```diff
@@ -8,11 +8,12 @@ import DateTime from '../../widgets/DateTime';
 import UsersNameContainer from '../../../containers/UsersNameContainer';
 import ExercisesNameContainer from '../../../containers/ExercisesNameContainer';
 import Icon, { EditIcon, CodeIcon } from '../../icons';
+import EnvironmentsListItem from '../../helpers/EnvironmentsList/EnvironmentsListItem';
 
-const ReferenceSolutionStatus = ({ description, solution: { userId, createdAt }, exerciseId }) => (
+const ReferenceSolutionStatus = ({ description, userId, submittedAt, submittedBy, exerciseId, environment }) => (
```
Comment (SemaiCZE) [inline src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js]: This comes with any API change?

Comment (Neloop) [reply src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js]: No, the solution structure was created in component above...

File: src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js (hunk 1 above)
Comment (Neloop) [reply src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js]: Copied from `SolutionStatus` component... Probably can be omitted.

Comment (krulis-martin) [reply src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js]: I think it was just defensive programming.

File: src/locales/cs.json [hunk 3]
```diff
@@ -965,6 +965,7 @@
   "app.randomMessages.m9": "Nakreslit hrocha",
   "app.referenceSolution.exerciseBroken": "Úloha je rozbitá, a proto není v tuto chvíli možné nechat toto vzorové řešení znovu vyhodnotit.",
   "app.referenceSolution.exerciseNoLongerHasEnvironment": "Úloha již nepodporuje běhové prostředí, pod kterým bylo toto řešení odevzdáno. Řešení není možné znovu vyhodnotit.",
+  "app.referenceSolution.reevaluatedBy": "Nechal znovu vyhodnotit",
```
Comment (krulis-martin) [inline src/locales/cs.json]: This is not gender-balanced. At least write 'Nechal(a)', or rewrite to something like 'Znovu vyhodnoceno uzivatelem'.

Comment (Neloop) [reply src/locales/cs.json]: Again... copied... will change

Comment (krulis-martin) [reply src/locales/cs.json]: OK, change the source as well, or better yet, merge the captions if they are really the same.

**Pull request merged**